pkg/controller/sync/sync_controller.go

```diff
@@ -147,17 +121,16 @@ func (r *ReconcileSync) Reconcile(ctx context.Context, request reconcile.Request
 		return reconcile.Result{}, nil
 	}
 
-	syncKey := r.metricsCache.GetSyncKey(unpackedRequest.Namespace, unpackedRequest.Name)
-	reportMetrics := false
+	reportMetricsForRenconcileRun := false
```

Comment:
`reportMetricsForReconcileRun` is wordy, we should leave as-is

Reply:
done